packages/traceloop-sdk/traceloop/sdk/utils/__init__.py

```diff
@@ -12,3 +12,15 @@ def camel_to_snake(s):
         return s.lower()
 
     return cameltosnake(s[0].lower() + s[1:])
+
+
+def is_notebook():
```

Comment:
[naming]

```suggestion
def should_process_single_spans():
```

Reply:
you can then have another method of `is_notebook` inside which is one case of single span processing. I just don't want to tie up the tracing code with notebook or not notebooks and other cases.

Reply:
I get it, but it shouldn't be in utils anyway IMO. I just created a separate method for initializing `span_processor`